### ISSUE TITLE
Can a python/pip requirements.txt file be added?

### ISSUE THREAD

**Issue by jaapcrezee** [armijnhemel/binaryanalysis-ng#25]
So you can easily set up a python virtual env? If you would consider merging I can create a fork.
Regards,

Jaap

**Comment by armijnhemel** [comment]
Yes, can be done, let me look into this. It might take me some time as I am a bit preoccupied with other things at the moment.

**Comment by jaapcrezee** [comment]
While you are at it, a `.gitignore` would also be nice.

**Comment by armijnhemel** [comment]
I just added a skeleton .gitignore file. I will be looking at the pip thing soon.

**Comment by eighthave** [comment]
The standard Python _setup.py_ should also be included while you're at it.  Then every Python dev and distro packager can automatically know how to install this project, without having to read docs or figure out custom layouts.

**Comment by timhemel** [comment]
Added a first `requirements.txt`, have only tested it in a docker install. It contains some development dependencies that you will not need when you just use the tool. It would be nice to separate them out, which is of course possible in a `setup.py`.

**Comment by timhemel** [comment]
Branch packagable includes a pyproject.toml, setup.cfg and setup.py.
